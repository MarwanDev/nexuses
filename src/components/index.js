export { default as Navbar } from './Navbar/Navbar';
export { default as Layout } from './Layout/Layout';
export { default as Header } from './Header/Header';
export { default as Footer } from './Footer/Footer';
export { default as DoctorSlider } from './DoctorSlider/DoctorSlider';
export { default as DoctorCard } from './DoctorCard/DoctorCard';
export { default as Table } from './Table/Table';

