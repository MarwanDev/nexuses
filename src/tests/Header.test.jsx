import { Header } from "../components";
import { BrowserRouter } from "react-router-dom";
import { describe, it, expect } from "vitest";
import renderer from "react-test-renderer";

describe("Header", () => {
  it("renders Header correctly", () => {
    const header = renderer
      .create(
        <BrowserRouter>
          <Header />
        </BrowserRouter>
      )
      .toJSON();
    expect(header).toMatchSnapshot();
  });
});
