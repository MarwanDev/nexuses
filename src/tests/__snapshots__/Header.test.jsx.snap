// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Header > renders Header correctly 1`] = `
<div
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#282D39",
      "borderRadius": 20,
      "display": "flex",
      "justifyContent": "space-between",
      "marginTop": "1.5rem",
      "padding": "2.5rem",
      "width": "100%",
    }
  }
>
  <h1
    className="health-header"
  >
    Health is Wealth
  </h1>
  <img
    alt="header"
    className="header-img"
    src="/src/assets/children.png"
    style={
      {
        "margin": "auto",
      }
    }
  />
</div>
`;

exports[`Navbar > renders navbar correctly 1`] = `
<div
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#282D39",
      "borderRadius": 20,
      "display": "flex",
      "justifyContent": "space-between",
      "marginTop": "1.5rem",
      "padding": "2.5rem",
      "width": "100%",
    }
  }
>
  <h1
    className="health-header"
  >
    Health is Wealth
  </h1>
  <img
    alt="header"
    className="header-img"
    src="/src/assets/children.png"
    style={
      {
        "margin": "auto",
      }
    }
  />
</div>
`;
